cpp/src/aztec/plonk/proof_system/widgets/random_widgets/permutation_widget_impl.hpp

```diff
@@ -380,22 +377,22 @@ barretenberg::fr ProverPermutationWidget<program_width, idpolys, num_roots_cut_o
     // (w_l(X) + β.σ_1(X) + γ).(w_r(X) + β.σ_2(X) + γ).(w_o(X) + β.σ_3(X) + γ).z(X).α
     // Once we divide by the vanishing polynomial, this will be a degree 3n polynomial. (4 * (n-1) - (n-4)).
 
-    std::array<const fr*, program_width> wire_ffts;
-    std::array<const fr*, program_width> sigma_ffts;
-    [[maybe_unused]] std::array<const fr*, program_width> id_ffts;
+    std::array<std::span<fr>, program_width> wire_ffts;
```

Comment:
can do `std::span<const fr>`

Reply:
Added `const` here and in many other locations